dokka-runners/dokka-gradle-plugin/src/main/kotlin/engine/parameters/DokkaGeneratorParametersSpec.kt

```diff
@@ -27,7 +27,7 @@ constructor(
     objects: ObjectFactory,
     /**
      * Configurations for Dokka Generator Plugins. Must be provided from
-     * [org.jetbrains.dokka.gradle.dokka.DokkaPublication.pluginsConfiguration].
+     * [org.jetbrains.dokka.gradle.engine.DokkaPublication.pluginsConfiguration].
```

Comment:
this KDoc link is unresolved (should be `formats` instead of `engine`) - the same in several other places

Reply:
Thanks! I will fix them.